src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
@powercode, your last commit had 7 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

SemanticVersion api tests.Error handling.Missing parts: 'NullString': ''
```powershell
Expected an exception, with FullyQualifiedErrorId 'PSArgumentNullException' to be thrown, but the FullyQualifiedErrorId was 'FormatException'. from /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1:206 char:15
    +             { [SemanticVersion]::new($version) } | Should -Throw -Err ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 206
206:             { [SemanticVersion]::new($version) } | Should -Throw -ErrorId $errorId
```
Alias tests.Export-Alias literal path.for CSV
```powershell
Expected <empty>, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1: line 57
57:                     $exportAliasError.FullyqualifiedErrorId | Should -Be $test.expectedError
```
Alias tests.Export-Alias literal path.for PS1
```powershell
Expected <empty>, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1: line 57
57:                     $exportAliasError.FullyqualifiedErrorId | Should -Be $test.expectedError
```
Alias tests.Export-Alias literal path.for Null
```powershell
Expected strings to be the same, but they were different.
Expected length: 95
Actual length:   102
Strings differ at index 32.
Expected: '...ErrorNullN...'
But was:  '...ErrorEmpty...'
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1: line 57
57:                     $exportAliasError.FullyqualifiedErrorId | Should -Be $test.expectedError
```
Alias tests.Import-Alias literal path.for Null
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.ImportAliasCommand' to be thrown, but the FullyQualifiedErrorId was 'ParameterArgumentValidationErrorEmptyStringNotAllowed,Microsoft.PowerShell.Commands.ImportAliasCommand'. from /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1:110 char:45
    +                 { Import-Alias -LiteralPath $test.testFile -ErrorActi ?
    +                                             ~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/alias.tests.ps1: line 110
110:                 { Import-Alias -LiteralPath $test.testFile -ErrorAction SilentlyContinue } | Should -Throw -ErrorId $test.expectedError
```